style.css

```diff
@@ -1,17 +1,66 @@
+body {
+  text-align: center;
+  display: flex;
+  justify-content: center;
+}
+
 .container h1,
 .container #books {
   text-align: center;
 }
 
 .flex {
+  background-color: #b3d1d2;
+  display: inline-block;
   display: flex;
   justify-content: center;
 }
 
 .btn {
   width: 20%;
+  background-color: #8e44ad;
+  color: #fff;
+  font-size: 18px;
+  display: inline;
+  float: right;
+  padding: 20px;
```

Comment:
- [ ] For the `add` button you will notice that from the wireframe given it is aligned to the right and does not cover the entire width of the form. Let's fix this. 💪 
![image](https://user-images.githubusercontent.com/68167810/129105618-614765c1-b3c4-464c-9ca0-2d5a8a8e6509.png)